adafruit_motorkit.py

```diff
@@ -296,3 +296,12 @@ def stepper2(self):
                 microsteps=self._steppers_microsteps,
             )
         return self._stepper2
+   
+    @property
+    def frequency(self):
+        """The overall PCA9685 PWM frequency in Hertz."""
+        return self._pca.frequency
+
+    @frequency.setter
+    def frequency(self, pwm_frequency=1600):
```

Comment:
i think remove the default value here?

Reply:
I like having the default in the setter when choosing values after instantiation so that I can quickly revert to the default without having to remember it. Is there a better way to accomplish that?

Now if I can only figure out how to pass the PR edit checks.

Also need to look at how to set PWM frequency in the Crickit library.

Reply:
ok thats fine!
you can run `black` to reformat
https://learn.adafruit.com/improve-your-code-with-pylint/black

Reply:
Thank you.

It passes `black` locally just fine. I'll get with Kattni to learn about why it fails during the build process.

<img width="923" alt="Screen Shot 2020-12-14 at 1 57 17 PM" src="https://user-images.githubusercontent.com/29906257/102140680-7b125880-3e14-11eb-8c36-0108dd54573e.png">

Reply:
dont use --check

Reply:
I'm in over my head on this, so I'll work with a mentor of mine. It's a great opportunity to learn a new skill.